internal/commands/app/create_inputs_test.go

```diff
@@ -209,14 +211,53 @@ func TestAppCreateInputsResolveDirectory(t *testing.T) {
 		assert.Equal(t, "new-app", inputs.LocalPath)
 	})
 
-	t.Run("should error when path specified is another realm app", func(t *testing.T) {
+	t.Run("should create default local directory name when FlagAutoConfirm is set", func(t *testing.T) {
```

Comment:
same thing here about capital letters in the test description
```suggestion
	t.Run("should create default local directory name when ui is set to auto confirm", func(t *testing.T) {
```